src/postprocessing/pp_command_output.rs

```diff
@@ -0,0 +1,178 @@
+use serde::Deserialize;
+use regex::Regex;
+
+
+#[derive(Deserialize)]
+pub struct CmdlineOutputFilter {
+    #[serde(default = "default_limit_lines")]
+    pub limit_lines: usize,
+    #[serde(default = "default_limit_chars")]
+    pub limit_chars: usize,
+    #[serde(default = "default_valuable_top_or_bottom")]
+    pub valuable_top_or_bottom: String,
+    #[serde(default = "default_grep")]
+    pub grep: String,
+    #[serde(default = "default_grep_context_lines")]
+    pub grep_context_lines: usize,
+    #[serde(default = "default_remove_from_output")]
+    pub remove_from_output: String,
+}
```

Comment:
I have no clue what it does and I will have no idea how to use it
what is remove_from_output?
what grep_context_lines is?
what if I don't want grep